community/index.md

```diff
@@ -8,9 +8,30 @@ Popular ways to connect with the Scala community include forums, chat rooms, loc
 
 The community is also the source of many libraries, tools, and other resources around Scala.
 
+## Who's behind Scala?
+
+The Scala language and associated websites are cooperative projects of
+the [Scala Center](https://scala.epfl.ch) at [EPFL](http://epfl.ch),
+[Lightbend](https://www.lightbend.com), and the Scala community
+more broadly, with participation from many companies, organizations,
+and individuals.
+
+The Scala team at Lightbend concentrates on advancing the language,
+compiler, and standard library.  The Scala Center focuses on education
+(especially online courses), documentation, open source community
+outreach, and tooling.  Community participation in all of these
+efforts is strongly encouraged.
+
+Scala was created by
+[Prof. Martin Odersky](http://lampwww.epfl.ch/~odersky/).  Martin and
+his research group at EPFL remain heavily involved with the language,
+including developing [Dotty](http://dotty.epfl.ch), a
+next-generation research compiler for Scala.
```

Comment:
I have pushed a commit omitting the word "research". That brings the wording in line with http://dotty.epfl.ch which simply says "A next generation compiler for Scala". I don't feel I should take @propensive's other suggestion unless/until the same change were also made on the Dotty site.

I understand that there is some confusion and uncertainty in the community about the status of Dotty, but there's only so much it's appropriate to do in this paragraph to address it. I think http://dotty.epfl.ch does a good job with this already, and about all we can do here is say a few words and give a link.